src/Workspaces/Remote/Razor/ServiceBase.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
```

Comment:
I played with this, and I would also have to add IVT to Workspace for `Checksum` and `Contract`. If that's ok then I can go ahead with this change.  I'd also want to ifdef the name so that we're putting the same class in two assemblies with the same name.

I'll add a commit that does this and you can let me know if you like it better that way.